pycozmo/brain.py

```diff
@@ -85,7 +85,8 @@ def on_behavior_done(self, cli: client.Client) -> None:
             self.deactivate_behavior()
 
     def on_cliff_detected(self, cli: client.Client, state: bool) -> None:
-        if state:
+        if state and not cli.robot_picked_up \
+           and (int(cli.left_wheel_speed.mmps) != 0 or int(cli.right_wheel_speed.mmps) != 0):
```

Comment:
The robot.RobotStatusFlag.IS_MOVING bit in the RobotStatus message may be better for this check.

Reply:
I used the ARE_TRACKS_MOVING instead because the IS_MOVING flag is also True for head lift movements.